Makefile

```diff
@@ -183,6 +232,16 @@ golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
 $(GOLANGCI_LINT): $(LOCALBIN)
 	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint,${GOLANGCI_LINT_VERSION})
 
+.PHONY: operator-sdk
+operator-sdk: ## Download operator-sdk locally.
+	@test -f $(OPERATOR_SDK) && echo "$(OPERATOR_SDK) already exists. Skipping download." && exit 0 ;\
+	echo "Downloading $(OPERATOR_SDK)" ;\
+        set -e ;\
+        mkdir -p $(dir $(OPERATOR_SDK)) ;\
+        OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+        curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk_$${OS}_$${ARCH} ;\
+        chmod +x $(OPERATOR_SDK)
```

Comment:
We should not have an operator-sdk target. 
From this repo perspective, the OLM artifacts are constant and hard-coded into the code base.